src/Photos.vue

```diff
@@ -165,7 +165,11 @@ export default {
 		if ('serviceWorker' in navigator) {
 			// Use the window load event to keep the page load performant
 			window.addEventListener('load', () => {
-				navigator.serviceWorker.register(generateUrl('/apps/photos/service-worker.js'), {
+				const webRootWithIndexPHP = getRootUrl() + '/index.php'
+				const doesURLContainIndexPHP = window.location.pathname.startsWith(webRootWithIndexPHP)
+				navigator.serviceWorker.register(generateUrl('/apps/photos/service-worker.js', {}, {
+					noRewrite: doesURLContainIndexPHP,
```

Comment:
```suggestion
					noRewrite: !doesURLContainIndexPHP,
```
we may need to invert the logic here to make it work. not sure though...

Reply:
or we simply always set it to true, WDYT?

Reply:
done